patches/@react-navigation__drawer@6.7.2.patch

```diff
@@ -0,0 +1,441 @@
+diff --git a/lib/module/views/modern/Drawer.js b/lib/module/views/modern/Drawer.js
+index 3aa1baa2efd65bab22da56416c90e3a294dab266..08fe2866924f233fa4bd349258a4b35ff51c7fc7 100644
+--- a/lib/module/views/modern/Drawer.js
++++ b/lib/module/views/modern/Drawer.js
+@@ -1,7 +1,38 @@
+-function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
```

Comment:
it looks like this patch got prettier'd, making the diff larger than it needed to be. if we disabled prettier for this, (1) it'd be easier to extend in the future, and (2) it'd be easier to merge upstream updates into this patch without conflict.

i can't tell the extent of the patch – is it small enough that redoing without prettier would be quick?

Reply:
good catch! yeah, it's a tiny patch, will fix.